core/src/test/scala/org/broadinstitute/dsde/rawls/entities/datarepo/DataRepoEntityProviderSpecSupport.scala

```diff
@@ -89,7 +89,7 @@ trait DataRepoEntityProviderSpecSupport {
 
   val defaultTables: List[TableModel] = List(
     new TableModel().name("table1").primaryKey(null).rowCount(0)
-      .columns(List("datarepo_row_id", "integer-field", "boolean-field", "timestamp-field").map(new ColumnModel().name(_)).asJava),
+      .columns(List("integer-field", "boolean-field", "timestamp-field").map(new ColumnModel().name(_)).asJava),
```

Comment:
this was the mock test data at fault; it returned "datarepo_row_id" in the snapshot description, whereas Data Repo does NOT return that in reality.